app/stateMachines/GetStartedStateMachine.js

```diff
@@ -311,12 +311,36 @@ export const getStartedMachine = Machine({
         }
       }
     },
+    // XXX
```

Comment:
comment still apply?

Reply:
Yeah, it does. 

I will come back to it later after release

Reply:
kk